src/ert/gui/model/snapshot.py

```diff
@@ -232,13 +230,13 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
                 if job.index:
                     job_node.data.index = job.index
                 if job.current_memory_usage:
-                    current_memory_usage = float(job.current_memory_usage)
-                    job_node.data.current_memory_usage = current_memory_usage
+                    current_memory_usage = int(float(job.current_memory_usage))
```

Comment:
do we need variable `current_memory_usage` here at all?

Reply:
We could instead do;

```
real_node.data.current_memory_usage = int(float(job.current_memory_usage))
self.root.data.current_memory_usage = int(float(job.current_memory_usage))
```

Reply:
I've updated the codebase accordingly.